internal/wasm/store_test.go

```diff
@@ -86,48 +85,84 @@ func TestModuleInstance_Memory(t *testing.T) {
 	}
 }
 
-func TestStore_AddHostFunction(t *testing.T) {
+func TestStore_NewHostModule(t *testing.T) {
```

Comment:
should we move this to host_test.go?

Reply:
hmm but these are code implemented in store

Reply:
I am wrong this code was moved (probably by me) to host.go I'll fix this!